src/Esfa.Vacancy.UnitTests/CreateApprenticeship/Application/GivenACreateApprenticeshipRequestValidator/AndLocationTypeOfOther/WhenValidatingAddressLine1.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+using Esfa.Vacancy.Application.Commands.CreateApprenticeship;
+using FluentValidation.TestHelper;
+using NUnit.Framework;
+
+namespace Esfa.Vacancy.UnitTests.CreateApprenticeship.Application.GivenACreateApprenticeshipRequestValidator.AndLocationTypeOfOther
+{
+    [TestFixture]
+    public class WhenValidatingAddressLine1
+    {
+        private static List<TestCaseData> TestCases() =>
+            new List<TestCaseData>
+            {
+                new TestCaseData(LocationType.OtherLocation, false, null, "31032").SetName("Then it is required"),
+                new TestCaseData(LocationType.OtherLocation, false, new string('a', 301),  "31033").SetName("Then cannot exceed 300 characters"),
+                new TestCaseData(LocationType.OtherLocation, false, "<p>", "31034").SetName("Then can include free text white list characters only"),
+                new TestCaseData(LocationType.OtherLocation, true, "10 Downing Street", null).SetName("Then it is valid"),
```

Comment:
Suggestion: `.SetName("And is in allowed format Then is invalid")`

And then do likewise for your other validator tests.